Osiris/Hacks/Backtrack.cpp

```diff
@@ -11,13 +11,13 @@ void Backtrack::update(FrameStage stage) noexcept
 {
     const auto localPlayer = interfaces.entityList->getEntity(interfaces.engine->getLocalPlayer());
     if (!config.backtrack.enabled || !localPlayer || !localPlayer->isAlive()) {
-        if (!records->empty())
-            records->clear();
+        for (auto& record : records)
+            record.clear();
 
         return;
     }
 
-    if (stage == FrameStage::NET_UPDATE_END) {
+    if (stage == FrameStage::RENDER_END) {
```

Comment:
Why do this in `FrameStage::RENDER_END`?

Reply:
that fixes the backtrack chams issue
sorry for not responding earlier